bulbs/notifications/viewsets.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import viewsets
+from rest_framework.permissions import IsAdminUser
+
+from bulbs.api.permissions import CanEditContent
+from .models import Notification
+from .serializers import NotificationSerializer
+
+
+class NotificationViewSet(viewsets.ModelViewSet):
```

Comment:
Does default view support filtering/searching per spec?

- ability to filter by active and inactive (draft included in active)
- search by internal name